README.md

```diff
@@ -1,2 +1,127 @@
 # devices
 Servo-specific APIs to access various devices
+
+----------
+
+###  How to use
+
+####Without the *bluetooth-test* feature:
+
+Each enum (`BluetoothAdapter`, `BluetoothDevice`, etc. ) will contain only one variant of the three possible default target. See the following `BluetoothAdapter` example:
```

Comment:
There are two target and one default target.
I think we should mention that if the target platform is unsupported, we fall back to our default (empty) implementation.